CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 4.6.0
+
+### Changed
+
+* Update to `NotificationsAPIClient.send_sms_notification()`
```

Comment:
Could match the email text below and substitute email => sms? But if the text is approved then its good.

Reply:
I've changed the text to match here